be/src/runtime/local_tablets_channel.cpp

```diff
@@ -1359,14 +1359,14 @@ void SecondaryReplicasWaiter::_send_replica_status_request(int unfinished_tablet
     }
     int num_unfinished_tablets = request.tablet_ids_size();
     _replica_status_closure->ref();
-#ifndef BE_TEST
-    stub->get_load_replica_status(&_replica_status_closure->cntl, &request, &_replica_status_closure->result,
-                                  _replica_status_closure);
-#else
+#ifdef BE_TEST
     std::pair<PLoadReplicaStatusRequest*, ReusableClosure<PLoadReplicaStatusResult>*> rpc_pair{&request,
-                                                                                               _replica_status_closure};
-    TEST_SYNC_POINT_CALLBACK("LocalTabletsChannel::rpc::get_load_replica_status", &rpc_pair);
+                                                                                           _replica_status_closure};
 #endif
+    TEST_SYNC_POINT_CALLBACK_OR_STMT("LocalTabletsChannel::rpc::get_load_replica_status", &rpc_pair, 
```

Comment:
This pattern is hard to understand, it takes the regular production code to be nested inside a macro which is purposely defined for unit test.